### PR TITLE
Adding French options 

### PR DESCRIPTION
**Pull request by AlxFrst**
- Added "FR" as a locale choice in index.js
- Updated locale handling in hello-fresh.js for "FR" locale

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const argv = yargs
     "Crawl on HelloFresh and use the GB locale"
   ).argv;
 
+
```
Comment (alexcodito) [inline index.js:30]: Please remove the redundant new line

File: services/hello-fresh.js [hunk 2]
```diff
@@ -147,6 +147,8 @@ const crawl = async function (settings) {
   if (settings.locale) {
     if (settings.locale === "DE") {
       apiSearchParams.locale = `de-${settings.locale.toUpperCase()}`;
+    } else if (settings.locale === "FR") {
+      apiSearchParams.locale = `fr-${settings.locale.toUpperCase()}`; // Gestion de la locale FR
```
Comment (alexcodito) [inline services/hello-fresh.js:151]: Please remove the comment in French

**Review by alexcodito — CHANGES_REQUESTED** [review]
Thank you for raising this PR. I've added a few comments.

**Comment by alexcodito** [comment]
Please also list FR in the README.md file.

**Review by alexcodito — CHANGES_REQUESTED** [review]
Please also list FR in the README.md file.

**Comment by AlxFrst** [comment]
Completed, sorry for errors :)

**Review by alexcodito — APPROVED** [review]

**Pull request merged**